components/pd_client/src/client_v2.rs

```diff
@@ -513,6 +513,39 @@ impl RpcClient {
     pub fn initialized(&self) -> bool {
         self.raw_client.initialized()
     }
+
+    fn handle_region_resp_by_id<Resp>(
+        &self,
+        region_id: u64,
+        mut func: impl FnMut(&mut pdpb::GetRegionResponse) -> Resp + Send + 'static,
```

Comment:
It's pointless to use both callback and future.

Reply:
I will fix this as well as `get_region_resp_by_id `